src/items/sxa-content/content/PLAY/playwebsite/Presentation/Page Designs.yml

```diff
@@ -35,7 +35,7 @@ SharedFields:
     </ruleset>
 - ID: "ba1f60d6-3deb-40cc-bb61-eec772279ee1"
   Hint: TemplatesMapping
-  Value: "%7bD8AA73E1-C275-4EEE-BB9D-333925E51952%7d%3d%257B5864BD63-B393-4F44-885D-C1242C5ABA5A%257D%26%7b819C5F1D-58E0-432F-BD5A-A523BA8AEBCE%7d%3d%257B4688E3B5-4981-443C-8480-C794EA691501%257D%26%7b3B884B30-54C1-464C-AADD-F8BD94E7C38E%7d%3d%257BAEE8FA57-7741-4F8A-9AD3-C7218491AE23%257D%26%7bADF8983F-D751-4AF2-804F-C9E3888F4219%7d%3d%257B348BC8E7-8B2B-419B-BAF0-F2D680F1E2E7%257D%26%7b4B8F13A2-EF1A-4A41-857D-A2191FD7C1E7%7d%3d%257BCB5FD20A-F30F-4FE0-A37B-6C01623CB5E5%257D%26%7b6DB09CEF-64A8-52A3-A980-38408109C4D7%7d%3d%257B5A35E289-C03B-4B9D-ACC4-7AE551CA1330%257D"
+  Value: "%7bD8AA73E1-C275-4EEE-BB9D-333925E51952%7d%3d%257B5864BD63-B393-4F44-885D-C1242C5ABA5A%257D%26%7b819C5F1D-58E0-432F-BD5A-A523BA8AEBCE%7d%3d%257B4688E3B5-4981-443C-8480-C794EA691501%257D%26%7b3B884B30-54C1-464C-AADD-F8BD94E7C38E%7d%3d%257BAEE8FA57-7741-4F8A-9AD3-C7218491AE23%257D%26%7bADF8983F-D751-4AF2-804F-C9E3888F4219%7d%3d%257B348BC8E7-8B2B-419B-BAF0-F2D680F1E2E7%257D%26%7b4B8F13A2-EF1A-4A41-857D-A2191FD7C1E7%7d%3d%257BCB5FD20A-F30F-4FE0-A37B-6C01623CB5E5%257D%26%7b6DB09CEF-64A8-52A3-A980-38408109C4D7%7d%3d%257B5A35E289-C03B-4B9D-ACC4-7AE551CA1330%257D%26%7b95CB88E7-8068-4A9C-9EEB-05F9A22C49D4%7d%3d%257B5A35E289-C03B-4B9D-ACC4-7AE551CA1330%257D"
```

Comment:
Oh, I think this is caused by the Settings "Templates location" value:
![image](https://github.com/Sitecore/Sitecore.Demo.XmCloud.PlaySummit/assets/2000200/f504ea4f-9fa5-4a49-957d-d1740e236f1b)

Since the legacy templates were moved from inside the PLAY folder to the Project folder, this mapping list rendering is broken.